{{ cookiecutter.project_name }}/setup.cfg

```diff
@@ -0,0 +1,11 @@
+[metadata]
+version = attr: {{ cookiecutter.package_name }}.__version__
+description = {{ cookiecutter.description }}
+long_description = file: README.md
```

Comment:
The default readme is kind of long and includes a bunch of stuff about the overall workflow (rather than about the specifics of using the python package).

Reply:
Ya this is true. Not sure what's a better option. I'd suggest future work to write up a `workflow.md` and make the `README.md` more targeted around the package's layout, install, etc.

Reply:
Maybe we just point this to a stub file, e.g. package.md which is minimally filled out with just the package name, etc.